File: testdata/jennies/rawtypes/field_with_struct_with_defaults/PHPRawTypes/src/Types/Defaults/NestedStruct.php
<?php

namespace Grafana\Foundation\Types\Defaults;

class NestedStruct implements \JsonSerializable {
    public string $stringVal;

    public int $intVal;

    /**
     * @return array<string, mixed>
     */
    public function jsonSerialize(): array
    {
        $data = [
            "stringVal" => $this->stringVal,
            "intVal" => $this->intVal,
        ];
        return $data;
    }
}

File: testdata/jennies/rawtypes/field_with_struct_with_defaults/CSharpRawTypes/src/Grafana/Foundation/Defaults/NestedStruct.cs
namespace Grafana.Foundation.Defaults;


public class NestedStruct
{
    public string StringVal;
    public long IntVal;

    public NestedStruct()
    {
        this.StringVal = "";
        this.IntVal = 0L;
    }

    public NestedStruct(string stringVal, long intVal)
    {
        this.StringVal = stringVal;
        this.IntVal = intVal;
    }
}
